src/community/clustering.jl

```diff
@@ -156,3 +156,24 @@ function global_clustering_coefficient(g::AbstractGraph)
     ntriangles == 0 && return 1.
     return c / ntriangles
 end
+
+"""
+    triangle_count(g::SimpleGraph)
```

Comment:
You are using `g::AbstractGraph` below, so you should write `triangle_count(g)` or `triangle_count(g::AbstractGraph)`.